src/helix-ui/styles/components/lists.less

```diff
@@ -1,24 +1,61 @@
 @import (reference) 'components/grid';
 
-hx-def + hx-def {
-  margin-top: @grid-gutter-size; // 20px
+dl {
```

Comment:
We need to add a class name qualifier. This CSS is applying to _every_ `<dl>` and it's children (regardless of if a consumer has opted in). To see an example of this issue, take a look at any of the Methods, Attributes, or Properties in the custom elements pages.

Let's go with `dl.hxList { ... }`